types/react-overlays/lib/RootCloseWrapper.d.ts

```diff
@@ -17,6 +23,4 @@ interface RootCloseWrapperProps {
 	event?: 'click' | 'mousedown';
 }
 
-declare class RootCloseWrapper extends React.Component<RootCloseWrapperProps> { }
-declare namespace RootCloseWrapper { } // module export workaround: https://github.com/Microsoft/TypeScript/issues/5073
-export = RootCloseWrapper;
+export default class RootCloseWrapper extends React.Component<RootCloseWrapperProps> { }
```

Comment:
Does this actually work correctly at runtime? When I originally wrote these a default export was not working.